src/components/shared/InfoPopover/index.ts

```diff
@@ -1 +1 @@
-export { default, Props } from './InfoPopover';
+export { default, Props } from './TokenInfoPopoverWrapper';
```

Comment:
Why is the `InfoPopover` index, exporting from `TokenInfoPopover`

Reply:
Not an issue now as I removed the InfoPopover